### DISCUSSION TITLE
There are problems with student certification

### DISCUSSION THREAD

**Discussion by braveKenynes5** [community/community#52480] [GitHub Education]
I am a student from China, apply for student certification on github recently, but Github official can not give the files (XUEXIN BAOGAO by Ministry of Education a certificate) passed, I have been tried many times but still failed, in this situation, which email should be contacted. thank you so muach for giving me some advice.

**Comment by samus-aran** [comment]
Hi @braveKenynes5 👋 

We do accept Ministry of Education certifications but we also suggest you provide additional pieces of evidence along side this.

Hope this helps! Let us know how you get along 👍
